phpunit/class-rest-sidebars-controller-test.php

```diff
@@ -407,4 +457,11 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'status', $properties );
 		$this->assertArrayHasKey( 'widgets', $properties );
 	}
+
+	public function users_without_permissions() {
```

Comment:
Are you sure this is working as expected? As far as I know, data providers are executed before setup before class, which means that the ids haven't been created yet. This is probably why the tests are passing when checking for a `401`.